### ISSUE TITLE
Create an Automation to Extract Invoice Details-Pre-requisites links are broken.

### ISSUE THREAD

**Issue by chaitanya-priya-puvvada** [sap-tutorials/sap-build-process-automation#60]
Tutorials: https://developers.sap.com/tutorials/spa-acquire-automation-store/spa-dox-create-automation.html
--------------------------

The pre-requisites links are broken 
![image](https://user-images.githubusercontent.com/91210196/226296201-77af65c5-1dcf-4fe1-b2c2-2dcd34d5f5ee.png)
Please check.

I checked by clicking the Invoice Approval process in Pre-requisites and noticed that the URL is generating very weirdly and everything looks fine in our repo.

![image](https://user-images.githubusercontent.com/91210196/226296750-3a568c2d-1030-487e-8413-0d979504f6f4.png)

**Comment by celineaudinsap** [comment]
@MichaelCzcz : https://developers.sap.com/tutorials/spa-acquire-automation-store/spa-dox-create-automation.html
This is a link that by mistake was promoted in a post on linkedin but still navigates to spa-dox-create-automation tutorial. Can you have a look?

**Comment by celineaudinsap** [comment]
@jmmargo: FYI

**Comment by celineaudinsap** [comment]
@jmmargo Yes the link is incorrect, and should not navigate to the tutorial but it does. I just wanted to bring that to your attention.

**Comment by jmmargo** [comment]
I just accessed this tutorial: https://developers.sap.com/tutorials/spa-dox-create-automation.html.
Then I clicked in the [Invoice Approval Process](https://developers.sap.com/tutorials/spa-dox-create-process.html) link in the prerequisites.  The correct tutorial opened for me.

I had the same experience with the [Install and Setup the Desktop Agent](https://developers.sap.com/tutorials/spa-setup-desktop-3-0-agent.html) link.  No issues when opening them.
